frontend/src/components/ParticipantInfo.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable indent */
```

Comment:
why?

Reply:
Weird and annoying linting errors I haven't devoted time to rectify. Mostly around tabs and spacing.

Reply:
Thought about adding something like prettier as part of the project?